corehq/apps/reports/models.py

```diff
@@ -655,6 +655,7 @@ class ReportNotification(CachedCouchDocumentMixin, Document):
     day = IntegerProperty(default=1)
     interval = StringProperty(choices=["daily", "weekly", "monthly"])
     uuid = StringProperty()
+    start_date = DateProperty(default=None)
```

Comment:
When is this set?

Reply:
https://github.com/dimagi/commcare-hq/blob/8784841cb1681a7a66e30fd8738e0e83c196e371/corehq/apps/reports/views.py#L993
This adds all the fields from `self.scheduled_report_form` into `self.report_notification`

Reply:
gotcha, that makes a lot more sense.  Thanks